### DISCUSSION TITLE
Suggestion: 24h time format

### DISCUSSION THREAD

**Discussion by PerHassel** [electricitymaps/electricitymaps-contrib#2757] [Ideas] [answered]
It would be easier to use electricitymap for persons not from US/UK if it was possible to select 24 hour clock on the timescales.

**Comment by Kongkille** [comment]
Hey, @PerHassel I would tend to agree. the clock also changes depending on the language but yes, as English is the default, it should perhaps be a 24-hour clock. 

WDYT @FelixDQ

**Reply by systemcatch** [reply]
Maybe we could have a 12/24hr checkbox next to the colour blind one?

**Reply by skovhus** [reply]
@systemcatch I like that. 👍

**Comment by VIKTORVAV99** [comment, marked as answer]
With the recent changes to the front-end the time format is now based on the standard defined for the language that is used.

It's still a 12 hour clock for English but other languages now have "native" versions of the clock and dates.

Perhaps not exactly what you wanted but at least it's standardized to the language used now.
